Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/properties/PropertyFileDAO.java

```diff
@@ -217,4 +228,7 @@ private Properties deepCopyProperties(PropertiesConfiguration properties) {
         return oRetProps;
     }
 
+    private static String getErrorMessage(String propertyFileName, String propertyName) {
+        return "Could not find the property: " + propertyName + " in the file:" + propertyFileName;
```

Comment:
can we use messageFormat instead of string-concatination?

`String result = MessageFormat.format(
     "At {1,time} on {1,date}, there was {2} on planet {0,number,integer}.",
     planet, new Date(), event);`

Reply:
Use StringBuilder instead